schema_utils.py

```diff
@@ -169,6 +174,17 @@ class Normalizers(object):
 
     @classmethod
     def get(cls, normalizer_id):
+        """Gets the value of the normalizer_id of the object.
```

Comment:
This docstring is wrong, I think. The normalizer ID is passed in, so you're not "getting" its value.

Most of the other parts of this docstring are too vague and need to be clearer. One thing you can try is to show this docstring to another developer (e.g. @vibhor98, but anyone else is fine too) and see if they understand what it means without looking at the implementation.

Reply:
I don't think this comment was addressed? I still think this docstring is wrong for the reasons given above.

Also, in the "Args" section, it is unclear what "attribute" refers to. I would just say that the normalizer_id is the name of the normalizer method that should be retrieved. That's what an "outsider" to the class would understand it as.